docs/lifecycle/lfn_entry/opx.md

```diff
@@ -1,2 +1,287 @@
-== OpenSwitch(OPX) Project Data ==
-**website** www.openswitch.net
+# OpenSwitch (OPX) Project Data
+
+## Instructions
+
+This document is a template to collect the data LFN thinks are relevant to LFN Project health. Projects will copy this template and instantiate it with their data as a part applying for lifecycle state transitions.
+
+`mkdir -p <relevant lifecycle transition dir>/<project name>/ cp project_data_template.rst !$/project_data.rst`
+
+If the project has instantiated this template before they are welcome to start from that base and update it with any new information. Be sure that the data requested by the primary copy of the Project Data Template has not changed.
+
+These instructions should be removed from the instantiated template.
+
+Here is the file for Tungsten Fabric. It’s a work in progress too: 
+
+[https://gerrit.linuxfoundation.org/infra/gitweb?p=lfn%2Fprocess.git;hb=refs%2Fchanges%2F80%2F12880%2F1;f=docs%2Flifecycle%2Flfn_entry%2Ftungsten_fabric%2Fproject_data.rst](https://gerrit.linuxfoundation.org/infra/gitweb?p=lfn%2Fprocess.git;hb=refs%2Fchanges%2F80%2F12880%2F1;f=docs%2Flifecycle%2Flfn_entry%2Ftungsten_fabric%2Fproject_data.rst)  
+
+[https://gerrit.linuxfoundation.org/infra/#/c/12880/36/docs/lifecycle/lfn_entry/tungsten_fabric/project_data.rst](https://gerrit.linuxfoundation.org/infra/#/c/12880/36/docs/lifecycle/lfn_entry/tungsten_fabric/project_data.rst) 
+
+## Project Data
+
+Data that describes a Linux Foundation Networking Project. Provide links to authoritative public content or steps to reproduce results when possible.
+
+### Project Vitals
+#### Project name: OpenSwitch (OPX)
+#### Project creation date:  June 2016
+#### Project license: Apache License, Version 2.0 
+#### Project release schedule 
+* History of at least two years or age of project 
+    * OPX 2.1.0 – 2017/07/31
+    * OPX 2.2.0 – 2018/02/02
+    * OPX 2.3.0 – 2018/03/28
+    * OPX 2.3.1 – 2018/05/14
+    * OPX 3.0.0 – 2018/09/07
+    * OPX 3.1.0 – 2018/12/20
+* Planned future release schedule
+    * OPX 3.2.0 – TBD
```

Comment:
5 releases in 2018 but none in 2019?

Reply:
The release of OPX is tied to OS10. We slowed down to 2 releases per year.